### DISCUSSION TITLE
Disconnect between Laptop and Phone viewing

### DISCUSSION THREAD

**Discussion by WadeRolfe** [stayonhire/feedback#31] [Feedback / Issues]
Hi team,

We have noticed a disconnect between our techs using their phones that dont show upcoming hires where the data and detail in the laptop system is there. When the team check on the "Upcoming hires" tab on their phones, the current job scheduled for today doesn't appear yet it is in the laptop showing as ready.

Is this a bug that you can fix or is there something we are missing?

![StayOnHire Mobile](https://github.com/stayonhire/feedback/assets/141391069/39eba687-253b-48c1-a3de-a8decbba7e4b)

![StayOnHire Laptop](https://github.com/stayonhire/feedback/assets/141391069/b7270a81-c545-40f0-a71b-78580160e196)

**Comment by twrayden** [comment]
Thanks Wade we've confirmed that is a bug, we're working on a fix now, and will let you know once it's complete.

**Comment by dylanhru** [comment]
Hi @WadeRolfe,

We're working on the fix but it's taking a little longer than expected. In short, it's to do with a few changes we made to the Field Actions app recently, and instead of 'bandaid' patch-fixing it, we're going to bring the mobile app up to full speed with recent upgrades to our architecture (across both the desktop and mobile codebases).

We are adding menu navigation throughout the mobile app and going to group assets by Date > Job > Time (instead of only Date > Time) in the past.
![Screenshot 2023-10-13 at 3 56 55 pm](https://github.com/stayonhire/feedback/assets/34927957/713a4a30-f6f0-45ff-89bc-a758b440f1f9)

'Asset' views are going to be separated into more clearly defined filters in a separate page.
![Screenshot 2023-10-13 at 4 00 10 pm](https://github.com/stayonhire/feedback/assets/34927957/3edb796d-b43b-4777-8f11-01bff112bf61)

Doing this right will allow us to bring the rest of StayOnHire's desktop functionality into the mobile app more easily and rapidly. For example, we want to bring Tasks (and later notifications, work orders and service tasks)
![Screenshot 2023-10-13 at 4 06 28 pm](https://github.com/stayonhire/feedback/assets/34927957/7065a735-8e7b-41f1-8de8-f92999f8c965)

While we're not going to get this done within the day as planned, the goal is to get the Date > Job > time functionality for your techs deployed by close of business Monday 16th Oct 2023 or earlier if we can.

This means Tues morning you should get an update in the app for changes to become visible.

**Comment by WadeRolfe** [comment]
Thanks guys, appreciated.

**Comment by dylanhru** [comment]
Hi @WadeRolfe, there has been an update to the mobile app now (though this is not bug-free yet). It was a little harder than expected but we plan on polishing all the kinks out tomorrow so keep an eye on your app as the day progresses and you should see it come along nicely.

**Comment by dylanhru** [comment]
Hi @WadeRolfe, the app is finally ready for you to get stuck back into. You would have seen fixes gradually deploying over the last week. We do have more quality-of-life improvements coming tomorrow (for To-Dos and the main navigation) and next week as we get more development time.

In summary, the app has been re-branded as, StayOnHire Field Actions, and is going to be heavily centred around getting data from the field into the office.

![IMG_99F66793710E-1](https://github.com/stayonhire/feedback/assets/34927957/ef811f4b-f7df-45be-864f-9ef4af745f84)

![IMG_5A2BCDF3F14E-1](https://github.com/stayonhire/feedback/assets/34927957/15664ebe-ea89-44d6-afb1-478f81e8c2d7)

![IMG_0397](https://github.com/stayonhire/feedback/assets/34927957/242c0474-4839-4f14-bcf2-78d4f0619af3)

![IMG_0398](https://github.com/stayonhire/feedback/assets/34927957/09e0e8d1-2bae-44ec-8738-9aa7650baf10)

![IMG_0399](https://github.com/stayonhire/feedback/assets/34927957/7a0798ab-c42e-4a01-a571-9095f815b5c4)

do please let me know if any issues arise from this point onwards!

**Comment by WadeRolfe** [comment]
Thanks for that Dylan, looks comprehensive.

Can you tell me how to get the app as I cant see the same set up in my phone as opposed to the other guys. I must have missed something somewhere.

**Comment by WadeRolfe** [comment]
Dont mind me, just got it from your webpage
